src/styles/layout/index.md.njk

```diff
@@ -39,7 +56,7 @@ Most GOV.UK pages follow a two-thirds to one-third layout but the grid system al
 
 Your main content should always be in a two-thirds column even if you’re not using a corresponding one-third column for secondary content.
 
-## Understanding the grid system
+### Understanding the grid system
 
 The grid is structured with a `govuk-o-grid` wrapper which acts as a row to contain your grid columns.
```

Comment:
not relevant to this PR, but just while it's in my mind - if we explain this as being a 'row', might it be clearer to to call it that in the name, eg `govuk-grid-row`. This would also align with CSS naming I think?

Reply:
Logged in the "Making things easier" doc